arenadata/readme.md

```diff
@@ -19,12 +23,55 @@ CI pushes docker images to the internal registry for each branch. We can pull it
 ## Full regression tests suite run
 
 We need to execute [../concourse/scripts/ic_gpdb.bash](../concourse/scripts/ic_gpdb.bash) in container to create demo cluster and run different test suites against it:
+for Ubuntu:
+```bash
+ docker run --name gpdb7_opt_on --rm -it -e TEST_OS=ubuntu \
+  -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=on' installcheck-world" \
+  --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_u22:latest \
+  /home/gpadmin/gpdb_src/concourse/scripts/ic_gpdb.bash
+```
+for Rocky Linux:
+```bash
+ docker run --name gpdb7_opt_on --rm -it -e TEST_OS=centos \
+  -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=on' installcheck-world" \
+  --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_regress:latest \
+  /home/gpadmin/gpdb_src/concourse/scripts/ic_gpdb.bash
+```
```

Comment:
Explain the changes compared to the original commit. Maybe it should be called gpdb8?